config/config.yaml

```diff
@@ -97,12 +97,6 @@ branch-protection:
             contexts:
             - pull-trust-manager-verify
             - pull-trust-manager-smoke
-        issuer-lib:
-          required_status_checks:
-            contexts:
-            - lint
-            - test-e2e
-            - test-unit
```

Comment:
question: I'm surprised we don't require _any_ checks here. I'd expect to still require the tests but for the names to change. Why are these removed?

Reply:
You only have to add these checks if they are non-prow checks.
prowjobs will be added automatically.

Reply:
That goes against what we're doing for other components (e.g. trust-manager above this)

Do you have some docs on that?

Reply:
https://docs.prow.k8s.io/docs/jobs/#posting-github-status-contexts
and
https://docs.prow.k8s.io/docs/jobs/#requiring-job-statuses

Reply:
I'm with you - I'd not noticed that before. The checks are required because they have `always_run: true` and `optional` defaults to `false`.

So we _could_ clean up a bunch of the other required_status_checks in this file! Not required for this PR of course.